index.js.flow

```diff
@@ -1,6 +1,6 @@
 // @flow
-import * as React from 'react';
-import type { ViewProps } from 'ViewPropTypes';
+import React, { type ElementConfig } from 'react';
```

Comment:
can you please keep the old import (import * as React from 'react';). Thanks.

it's because of https://flow.org/en/docs/react/components/

> Note: We import React as a namespace here with import * as React from 'react' instead of as a default with import React from 'react'. When importing React as an ES module you may use either style, but importing as a namespace gives you access to React’s utility types.